deploy/helm/values.yaml

```diff
@@ -190,6 +190,27 @@ otel:
 
           # Timeout if metrics can't be retrieved in given time interval
           scrapeTimeout: 25s
+    
+    # k8s_instrumentation controls the automatic extraction of Kubernetes metadata from resources.
+    # It instruments OpenTelemetry (OTEL) resources being sent.
+    k8s_instrumentation:
+      labels:
+        # Set 'enabled' to true to instrument Kubernetes labels.
+        enabled: true
+
+        # Provide a regular expression pattern to exclude specific labels from instrumentation.
+        # Example: To exclude labels with 'internal' or 'private' in their names, use the following pattern:
+        # excludePattern: ".*internal.*|.*private.*"
+        excludePattern:
+
+      annotations:
+        # Set 'enabled' to true to instrument Kubernetes annotations.
+        enabled: false
```

Comment:
Is this disabled just for events or also for metrics?

Reply:
We need annotations just to display its list for entities in UI, it won't be used for filtering, so we don't need to send them with both events and metrics. So it is enabled by default on metrics and disabled on events (to reduce data).